src/content/basics/azure-architecture/index.md

```diff
@@ -14,29 +14,29 @@ Giant Swarm's Architecture is split into two logical parts, one being the Contro
 
 ## Giant Swarm control plane
 
-![Control Plane Architecture](/img/architecture-azure-control-plane.png)
+![Control Plane Architecture](architecture-azure-control-plane.png)
 
 The above diagram shows our control plane setup running with two bastion hosts in different availability zones. All instances are in a private subnet and only accessible through the bastion hosts. Bastions hosts are available via Azure VPN connection only.
 
-The cluster has several APIs and Interfaces. The Kubernetes API of the Control Plane is only available to Giant Swarm operations personnel and only through the Bastion hosts. 
+The cluster has several APIs and Interfaces. The Kubernetes API of the Control Plane is only available to Giant Swarm operations personnel and only through the Bastion hosts.
```

Comment:
Is it "Bastions hosts", "Bastion hosts" ?
It's not consistent in the document.

Reply:
changed to 'Bastion hosts' everywhere